clients/grpc-client/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2020 Dremio
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.projectnessie</groupId>
+    <artifactId>nessie-clients</artifactId>
+    <version>0.9.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>grpc-client</artifactId>
+
+  <name>Nessie - Client - GRPC</name>
+
+  <dependencies>
```

Comment:
Having the code in this artifact in `:nessie-client` should also help to revert the `nessie-client` artifact-dance in the (spark-)tests

Reply:
as mentioned in the other comment, I think http client + grpc client code should stay in their own modules

Reply:
fwiw, moving the grpc client stuff into `nessie-client` doesn't fix the circular dependency issue that we have (where delta/iceberg can depend on potentially older versions of the nessie-client)

Reply:
Http isn't in its own module, no?

Reply:
not sure I follow, can you elaborate what you mean exactly?

Reply:
You said: "_I think http client + grpc client code should stay in their own modules_" - but http is not in its own module - it's in `:nessie-client` - not in `:nessie-client-http` analogous to `:nessie-client-grpc`.

Reply:
my assumption was that the current `nessie-client` == http client, therefore it's kind of in its own module, but intertwined with the general client APIs (since there was only one client implementation). I don't have a strong preference where the grpc client-code should live. Let's see what @rymurr thinks.